src/components/WebexActivityStream/WebexActivityStream.scss

```diff
@@ -1,3 +1,71 @@
+// For Storybook, so that activity stream loads content properly
+#root {
```

Comment:
why don't you put this at the top level? this might get lost here. maybe `scss` file to be imported into setupFile

Reply:
I tried that, doesn't seem to work.